build_runner/lib/src/generate/build.dart

```diff
@@ -121,8 +121,9 @@ Stream<BuildResult> watch(List<BuildAction> buildActions,
 /// By default a static server will be set up to serve [directory] at
 /// [address]:[port], but instead a [requestHandler] may be provided for custom
 /// behavior.
-Stream<BuildResult> serve(List<BuildAction> buildActions,
+Future<BuildHandler> buildHandler(List<BuildAction> buildActions,
```

Comment:
I am not sure about the name here, it sounds more like its handling builds than requests :).

Reply:
Change to ServeHandler and dropped this method in favor of having only `watch`